utils/joe/files/joerc

```diff
@@ -261,6 +328,7 @@ hnext		^[ .
 hprev		^[ ,
 math		^[ m
 math		^[ M
+ math		^[ ^M
```

Comment:
Same also here. several of the additions seem to contain a leading space. Looks strange.